### DISCUSSION TITLE
Behaviour of wildcard certificates (i guess)

### DISCUSSION THREAD

**Discussion by giveerr** [timothystewart6/launchpad#23] [Questions] [answered]
Hey tim.

I'm trying to create my own setup with your "Wildcard Certificates with Traefik + cert-manager" video and I think I found an interesting behavior.

as you describe both cloudflare and configs have wildcard dns entry
for example: ***.example.com**

Everything works fine with 1st degree subdomains like "a.example.com" 
**But it breaks when using multi-level subdomains** like "a.b.example.com"

I spent hours trying to realize this. I would like to leave this painful experience as a note here, maybe you can consider adding it in your documentation.

(don't worry i know this is not related to you 😅)

**Comment by timothystewart6** [comment, marked as answer]
Thank you!  Yeah, I think you have to generate a cert for each sub domain before going down another level!  Thank you!
